packages/components/addon/components/hds/app-footer/copyright.hbs

```diff
@@ -0,0 +1,9 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::AppFooter::Item class="hds-app-footer__item--copyright" ...attributes>
+  <FlightIcon @name="hashicorp" />
+  <Hds::Text::Body @tag="span" @size="100" @color="faint">© 2023 HashiCorp</Hds::Text::Body>
```

Comment:
Not sure if it's been discussed (maybe something to ask ambassadors?) but should the year be dynamic so it auto-updates (via JS)? No strong opinions, there are pros/cons in both cases.

Reply:
I added an argument for the year that uses the current year by default but can accept a custom year value if desired.